docs/developers/git-workflow.rst

```diff
@@ -754,14 +754,29 @@ Invenio git commit log messages are usually formatted in the following
 way:
 
 * commit message headline providing short summary (maximum 50 chars)
-  formatted in the style of ``ModuleName: short description``. (using mostly
-  nouns, no verbs);
+  formatted in the style of ``component: short description``. (using
+  mostly nouns, no verbs);
 
 * empty line;
 
-* commit message body with a detailed description of what this patch
-  does, formatted as a bulletted list, with one empty line between
-  items (using present tense).
+* commit message body with a detailed description of what this patch does,
+  formatted as a bulletted list, with one empty line between items (using
+  present tense), and leading label for bullets with *important* messages
+  that should be included in release notes.
```

Comment:
For me the labels are OK. Was just pointing out that,  we don't want to make developers to think that *every* commit line is going in the changelog, and that we are still interested into developers to contribute commit logs also if they are not going in the release notes.

So you can reformulate to something such as:
```
* commit message body with a detailed description of what this patch does,
  formatted as a bulletted list, with one empty line between items (using
  present tense). If a given element in the list should appear in the release 
  notes (to describe changes to Invenio admins and developers) it must be
  prefixed with one of the following labels: